noir-projects/noir-protocol-circuits/crates/public-kernel-lib/src/common.nr

```diff
@@ -363,7 +363,7 @@ fn propagate_new_nullifiers(
 fn propagate_new_l2_to_l1_messages(public_call: PublicCallData, public_inputs: &mut PublicKernelCircuitPublicInputsBuilder) {
     // new l2 to l1 messages
     let public_call_public_inputs = public_call.call_stack_item.public_inputs;
-    let portal_contract_address = public_call.portal_contract_address;
+    // let portal_contract_address = public_call.portal_contract_address;
```

Comment:
```suggestion
```

Why leave this commented out?

Reply:
:shrug: in case you guys wanted it,  I'm assuming it was wanted to be kept due to not being removed before.